### PR TITLE
Proposed landing page update

### PR DESCRIPTION
**Pull request by dabrahams**

### CODE REVIEW

File: index.md [hunk 1]
```diff
@@ -33,35 +33,38 @@ Here's a simple program:
 
 ```val
 subscript longer_of(_ a: inout String, _ b: inout String): String {
-  yield if b.count() > a.count() { &b } else { &a }
+  if b.count() > a.count() { yield &b } else { yield &a }
+}
+
+func emphasize(_ z: inout String, strength: Int = 1) {
+  z.append(repeatElement("!", count: strength)))
```
Comment (kyouko-taiga) [inline index.md:40]: I would prefer if we stick to snake_case in the standard library.

File: index.md [hunk 2]
```diff
@@ -33,35 +33,38 @@ Here's a simple program:
...
 }
 
 public fun main() {
   var (x, y) = ("Hi", "World")
-  inout z = longer_of[&x, &y]
-  z .append("!")
+  emphasize(longer_of[&x, &y])
```
Comment (kyouko-taiga) [inline index.md:45]: I'm afraid we need another another `&` here:
```val
emphasize(&longer_of[&x, &y])
```

**Review by kyouko-taiga — CHANGES_REQUESTED** [review]
See my two comments.

File: index.md (hunk 1 above)
Comment (dabrahams) [reply index.md:40]: Sorry, just forgot.  Use of suggested change feature is generally appreciated when possible

Comment (dabrahams) [reply index.md:40]:
```suggestion
  z.append(repeat_element("!", count: strength)))
```

File: index.md (hunk 2 above)
Comment (dabrahams) [reply index.md:45]:
```suggestion
  emphasize(&longer_of[&x, &y])
```

**Review by kyouko-taiga — APPROVED** [review]

**Pull request merged**